Misc/NEWS.d/next/Library/2025-12-16-14-49-19.gh-issue-142783.VPV1ig.rst

```diff
@@ -0,0 +1,3 @@
+Fix zoneinfo use-after-free with descriptor _weak_cache. Return new
+reference from get_weak_cache() instead of borrowed reference to prevent
+premature object free.
```

Comment:
This describes implementation details (C function names) which are not visible to the end user. Please only describe the user visible change.